src/components/Notifications/NotificationsFeed.tsx

```diff
@@ -52,9 +60,20 @@ export default function NotificationsFeed({
 }: Props) {
   const t = useFormatMessage()
   const [isSubscribing, setIsSubscribing] = useState(false)
+  const [isDiscordConnect, setIsDiscordConnect] = useState(false)
   const [user, userState] = useAuthContext()
   const [notificationsPerPage, setNotificationsPerPage] = useState(NOTIFICATIONS_PER_PAGE)
   const lastNotificationIdIndex = userNotifications?.findIndex((item) => item.payload_id === lastNotificationId)
+  const { isProfileValidated: isValidatedOnDiscord, refetch } = useIsProfileValidated(user, [AccountType.Discord])
+  const [isDiscordChanging, setIsDiscordChanging] = useState(false)
+  const [isSettingsOpened, setIsSettingsOpened] = useState(false)
+  const { isDiscordActive, refetch: refetchIsDiscordActive } = useIsDiscordActive()
+
+  useEffect(() => {
+    if (isOpen && !isValidatedOnDiscord) {
+      refetch()
```

Comment:
Could you change this function name to `refetchIsProfileValidated`?

Also what use case is covering this effect?